package.json

```diff
@@ -2,29 +2,31 @@
   "name": "@veupathdb/eda",
   "version": "0.14.11",
   "dependencies": {
-    "@emotion/babel-preset-css-prop": "^11.2.0",
     "@emotion/react": "^11.4.1",
     "@emotion/styled": "^11.3.0",
     "@material-ui/core": "^4.11.3",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.58",
     "@mui/styles": "^5.0.1",
     "@types/debounce-promise": "^3.1.3",
-    "@veupathdb/components": "^0.8.8",
     "@veupathdb/core-components": "^0.2.43",
+    "@veupathdb/web-common": "^0.1.10",
+    "@veupathdb/components": "^0.8.8",
     "@veupathdb/http-utils": "^1.0.1",
     "@veupathdb/study-data-access": "^0.0.3",
     "@veupathdb/wdk-client": "^0.4.9",
     "debounce-promise": "^3.1.2",
     "file-saver": "^2.0.5",
     "fp-ts": "^2.9.3",
     "io-ts": "^2.2.13",
-    "lodash": "^4.17.20",
+    "lodash": "^4.17.21",
     "monocle-ts": "^2.3.11",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-draggable": "^4.4.3",
     "react-resizable": "^3.0.4",
+    "react-router": "^5.2.1",
```

Comment:
I think only `react-router-dom` is needed.

```suggestion
```

Reply:
I see a number of places using imports from react-router.

For ex. see `MapVeuContainer.tsx` or `AnalysisPanel.tsx`